wall_e/extensions/ban.py

```diff
@@ -340,9 +346,6 @@ async def ban(self, ctx, user: discord.Member, *args):
 
         self.logger.info(f"[Ban ban()] User kicked from guiled at {dt}.")
 
-        # begin purging messages from user in the last purge_window_days
```

Comment:
what made you decided to move this below the db modification and why is that not a good reason to move anything else below?

Reply:
i wanted purge messages to happen basically last, so mod's got notified in mod_channel that the ban was done. this is the soft issue that caused the last duplication, the command didn't finish/provide feedback so the mod assumed the command didn't work and did a guild ban causing an intercept.

Reply:
ah, gotcha